docs/docs/rest-api/public/api/v2/apps.raml

```diff
@@ -0,0 +1,451 @@
+
+###
+# Get a list of applications
+#
+get:
+  description: Get the list of running applications. Several filters can be applied to filter the result list.
+  is: [ secured ]
+  queryParameters:
+    cmd:
+      description: the command of this application
+      example: java
+    id:
+      description: the id or part of the id of the application
```

Comment:
"Filter the result to only return apps whose `id` is or contains the given value"